src/modules/landing_target_estimator/landing_target_estimator_params.c

```diff
@@ -187,3 +187,51 @@ PARAM_DEFINE_FLOAT(LTEST_SENS_POS_Y, 0.0f);
  *
  */
 PARAM_DEFINE_FLOAT(LTEST_SENS_POS_Z, 0.0f);
+
+/**
+ * Euler Angle Rotation (3-2-1 intrinsic Tait-Bryan) Phi (X)
+ *
+ * Orientation transformation from camera frame that detected
+ * the landing_target to drone's NED frame
+ *
+ * @unit rad
+ * @decimal 2
+ *
+ * @group Landing target Estimator
+ */
+PARAM_DEFINE_FLOAT(LTEST_EULR_PHI, 0.0f);
+
+/**
+ * Euler Angle Rotation (3-2-1 intrinsic Tait-Bryan) Theta (Y)
+ *
+ * Orientation transformation from camera frame that detected
+ * the landing_target to drone's NED frame
+ *
+ * @unit rad
+ * @decimal 2
+ *
+ * @group Landing target Estimator
+ */
+PARAM_DEFINE_FLOAT(LTEST_EULR_THETA, 0.0f);
+
+/**
+ * Euler Angle Rotation (3-2-1 intrinsic Tait-Bryan) Psi (Z)
+ *
+ * Orientation transformation from camera frame that detected
+ * the landing_target to drone's NED frame
+ *
+ * @unit rad
+ * @decimal 2
+ *
+ * @group Landing target Estimator
+ */
+PARAM_DEFINE_FLOAT(LTEST_EULR_PSI, -1.57f);
```

Comment:
Default yaw should be 0, no?

Reply:
These three angles are the exact settings that I needed to make it work with the data coming from the gazebo simulation, where Z is pointing to the sky. For PX4 I thus had to rotate it by 180 such that Z points into the ground.

Instead of defining the defaults here, I could define the angle also in gazebo's Iris airframe file, and default it to 0 in PX4 instead.